src/__tests__/companyDb.test.js

```diff
@@ -133,4 +134,41 @@ describe('Companies model tests', () => {
       expect(customerObj.companyIds).toContain(_company._id);
     }
   });
+
+  test('removeCompany', async () => {
+    const company = await companyFactory({});
+    await internalNoteFactory({
+      contentType: COC_CONTENT_TYPES.COMPANY,
+      contentTypeId: company._id,
+    });
+
+    const removed = await Companies.removeCompany(company._id);
+
+    expect(
+      await InternalNotes.find({
+        contentType: COC_CONTENT_TYPES.COMPANY,
+        contentTypeId: company._id,
+      }),
+    ).toHaveLength(0);
+    expect(removed.result).toEqual({ n: 1, ok: 1 });
+  });
+
+  test('mergeCompanies', async () => {
```

Comment:
Why are you not testing actual logics ?. Where is checks of merge company fields ?